cuqi/experimental/mcmc/_sampler.py

```diff
@@ -102,7 +102,7 @@ def step(self):
         pass
 
     @abstractmethod
-    def tune(self):
+    def tune(self, skip_len, update_count):
```

Comment:
Document what skip_len and update_count do.

Reply:
I added my understanding. I think we need to revisit these parameters. They are used due to MH, pCN etc. using them. But we may avoid these parameters and instead use internal state to track this.